### DISCUSSION TITLE
Working Nano Train and City

### DISCUSSION THREAD

**Discussion by EclecticInventor** [pybricks/support#1709] [Show and tell]
https://youtu.be/li9mg-D0mS0
[![video thumbnail](http://i3.ytimg.com/vi/li9mg-D0mS0/hqdefault.jpg)](https://youtu.be/li9mg-D0mS0)

**Comment by laurensvalk** [comment]
Looks great :)

**Reply by EclecticInventor** [reply]
Thank you for helping to make it possible!
